emails/views.py

```diff
@@ -396,7 +396,8 @@ def _get_domain_address(to_address, local_portion, domain_portion):
 def _get_address(to_address, local_portion, domain_portion):
```

Comment:
Could we include some tests for this `_get_address` function while we're in here?

Reply:
Done 8b54661